internal/controller/drcluster_mmode_test.go

```diff
@@ -98,14 +97,12 @@ var _ = Describe("DRClusterMModeTests", Ordered, func() {
 				Kind:       "RamenConfig",
 				APIVersion: rmn.GroupVersion.String(),
 			},
-			ControllerManagerConfigurationSpec: controller_runtime_config.ControllerManagerConfigurationSpec{
-				LeaderElection: &config.LeaderElectionConfiguration{
-					LeaderElect:  new(bool),
-					ResourceName: ramencontrollers.HubLeaderElectionResourceName,
-				},
-				Metrics: controller_runtime_config.ControllerMetrics{
-					BindAddress: "0", // Disable metrics
-				},
+			LeaderElection: &config.LeaderElectionConfiguration{
```

Comment:
Rest of the changes look good to me. However, similar change as this is required in tests wherever we start a new manager. Search for `ctrl.NewManager`

Reply:
@raghavendra-talur , yes, it is changed [here](https://github.com/ELENAGER/ramen/blob/7d893b743a8a8bdf2b1fd537658829efe0368e76/internal/controller/suite_test.go#L310), but not needed here[](https://github.com/ELENAGER/ramen/blob/7d893b743a8a8bdf2b1fd537658829efe0368e76/internal/controller/volsync/volsync_suite_test.go#L115)